WTC_Flight/Src/main.c

```diff
@@ -73,6 +73,8 @@
 #include "stateMachine/stateMachine.h"
 #include "PI_Control/SS_Pi_Comms.h"
 #include "PI_Control/commands.h"
+#include "stateMachine/comms.h"
+#include "mygpio/mygpio.h"

```

Comment:
Added these includes to fix the `implicit declaration of function` error for `commsFromGround`. (The `mygpio.h` include was actually accidentally and is unnecessary.)